ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -66,6 +66,17 @@ open class PopupMenuController: DrawerController {
         }
         return height
     }
+
+    open override var backgroundColor: UIColor {
+        get {
+            return super.backgroundColor
+        }
+        set {
+            super.backgroundColor = newValue
+            tableView.backgroundColor = newValue
```

Comment:
Any reason not to handle that with a didSet instead of overriding the getter and setter?